cmd/contour/serve.go

```diff
@@ -486,11 +486,9 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	}
 	g.Add(sh.Start)
 
-	// Now we have the statusUpdateWriter, we can create the StatusWriter, which will take the
+	// Now we have the statusUpdateWriter, we can create the event handler's StatusUpdater, which will take the
```

Comment:
nit: s/statusUpdateWriter/statusUpdateHandler

Related, a few lines above this the log context is still "StatusUpdateWriter", should now be "StatusUpdateHandler".